Sources/MongoSwift/MongoCursor.swift

```diff
@@ -197,37 +197,58 @@ public class MongoCursor<T: Codable>: Sequence, IteratorProtocol {
      *   - `DecodingError` if an error occurs decoding the server's response.
      */
     public func nextOrError() throws -> T? {
-        if let next = self.next() {
```

Comment:
(posting here as I can't comment on the actual line) this method's docstring references the old behavior of `next()`. we should update it to reflect your changes.